apps/game/templates/competition/detail.html

```diff
@@ -8,9 +8,12 @@
             <div class="mirror__item mrgb--">
                 <h1 class="mrgb0">{{ competition }}</h1>
 
-                {% if competition.description %}
-                    <p class="mrgv0 text-muted">{{ competition.description | linebreaksbr }}</p>
-                {% endif %}
+                <ul class="list-inline list-inline--divided text-muted">
+                    <li>{{ competition.sport.icon|safe }} {{ competition.sport }}</li>
+                    {% if competition.description %}
+                        <li>{{ competition.description }}</li>
```

Comment:
Yeah the new display as an inline list make it difficult to display long descriptions with line breaks. According to its current usage I’m feeling like the description should be limited in characters length and be more like a text field than a text area. WDYT?